core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/move.kt

```diff
@@ -51,39 +220,152 @@ public fun <T> DataFrame<T>.moveTo(newColumnIndex: Int, vararg columns: KPropert
 
 // endregion
 
-// region moveToLeft
+// region moveToStart
+
+/**
+ * ## The MoveToStart Operation
+ *
+ * Moves the specified [columns] to the [DataFrame] start.
```

Comment:
maybe specify that this means the top-level, or do you think that's clear enough from this?

Reply:
done